e2e/yaks/common/kamelet-beans/beans-source.kamelet.yaml

```
@@ -0,0 +1,36 @@
+# ---------------------------------------------------------------------------
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ---------------------------------------------------------------------------
+
+apiVersion: camel.apache.org/v1alpha1
+kind: Kamelet
+metadata:
+  name: beans-source
+  labels:
+    camel.apache.org/kamelet.type: "source"
+spec:
+  template:
+    beans:
+    - name: myBean
+      type: "#class:java.util.Date"
+      properties:
+        time: 0
```

```suggestion
      property:
      - key: time
        value: 0
```

Back to the ugly syntax...

---

Thanks! I will fix it :)